samples/corda/corda-simple-application/clients/src/main/kotlin/com/cordaSimpleApplication/client/UtilsManager.kt

```diff
@@ -0,0 +1,230 @@
+    }
+
+    return networksConfigJSON.getJSONObject(networkID)
+}
+
+/*
+ * Fetches the certificate corresponding to an user from the file 'networkID'+'_UsersAndCerts.json'.
+ * This is used during Pledge to get the recipientCert, and during Claim to get the pledgerCert.
+ */
+fun getUserCertFromFile(userID: String, networkID: String): String {
+    var certBase64: String
+    try {
+
+        val credentialPath: String = System.getenv("MEMBER_CREDENTIAL_FOLDER") ?: "clients/src/main/resources/config/credentials"
+        val dirPath: String = "$credentialPath" + "/../remoteNetworkUsers"
+        val filepath: String = "$dirPath/$networkID" + "_UsersAndCerts.json"
+
+        var usersAndCertsJSON: JSONObject
+        var usersAndCertsFile: File
+        usersAndCertsFile = File(filepath)
+        if (!usersAndCertsFile.exists()) {
+            // if file doesn't exits, throw an exception
+            println("File $filepath doesn't exist to fetch the certificate of user $userID.")
+            throw IllegalStateException("File $filepath doesn't exist to fetch the certificate of user $userID.")
+        } else {
+            // if file exists, read the contents of the file
+            usersAndCertsJSON = JSONObject(usersAndCertsFile.readText(Charsets.UTF_8))
+        }
+
+        val x500NameOfuserID = CordaX500Name.parse(userID).toString()
+        // throw exception if the userID is not present in the file
+        if (!usersAndCertsJSON.has(x500NameOfuserID)) {
+            println("File $filepath doesn't contain the certificate of user $userID.")
+            throw IllegalStateException("File $filepath doesn't contain the certificate of user $userID.")
+        }
+
+        certBase64 = usersAndCertsJSON.getString(x500NameOfuserID)
+    } catch (e: Exception) {
+        println(e.toString())
+        exitProcess(1)
+    }
+    return certBase64
+}
+
+/*
+ * Populates the file 'networkID'+'_UsersAndCerts.json' with the users and their certificates.
+ * This is used during Pledge to get the recipientCert, and during Claim to get the pledgerCert.
+ */
+class SaveUserCertToFileCommand : CliktCommand(name="save-cert", help = "Populates the file 'networkId' + '_UsersAndCerts.json' with the certificate of 'ourIdentity'")
+{
+    val config by requireObject<Map<String, String>>()
+    override fun run() {
+        println("Fetching base64 certificate of the user 'ourIdentity'.")
+        val rpc = NodeRPCConnection(
+            host = config["CORDA_HOST"]!!,
+            username = "clientUser1",
+            password = "test",
+            rpcPort = config["CORDA_PORT"]!!.toInt())
+        try {
+            val userID: String = rpc.proxy.nodeInfo().legalIdentities.get(0).name.toString()
+            val certBase64: String = fetchCertBase64Helper(rpc.proxy)
+            var networkID: String
+            val cordaPort: Int = config["CORDA_PORT"]!!.toInt()
+            if (cordaPort == 30006) {
+                networkID = "Corda_Network2"
```

Comment:
Can we use Env var `NETWORK_NAME` instead of hard coding based on Port? This will solve the issue me and Rama were facing.

Reply:
Done.